### ISSUE TITLE
module test fails

### ISSUE THREAD

**Issue by mattvenn** [etalian/mensa.wip#3]
![image](https://user-images.githubusercontent.com/634220/122044386-ba7a2b80-cddc-11eb-8106-2f43baaeaef6.png)

forgot to commit testfloat_gen ?

**Comment by etalian** [comment]
Indeed, it is a third party tool from Berkeley, unfortunately setting it up requires a handful of extra steps, that are now documented in [.gitlab-ci.yml](../blob/ce99aec5fb7994d8998f9e1eaba563d99c172a0b/.gitlab-ci.yml#L14) (under test-module)

Once set up, the test passes:
https://gitlab.com/etalian/mensa.wip/-/jobs/1349375418
